Public/Src/Engine/Processes/Containers/ContainerConfiguration.cs

```diff
@@ -41,18 +41,36 @@ public sealed class ContainerConfiguration
         /// </remarks>
         public IReadOnlyDictionary<AbsolutePath, IReadOnlyList<ExpandedAbsolutePath>> OriginalDirectories { get; }
 
+        /// <summary>
+        /// On Windows, defines whether the WCI (Windows Container) filter driver is enabled. When false,
+        /// only BindFlt is used for output redirection.
+        /// </summary>
+        public bool IsWciFilterEnabled { get; }
+
+        /// <summary>
+        /// Paths that should not have the bindflt path transformation applied to them.
+        /// </summary>
+        public IReadOnlySet<string> BindFltExcludedPaths { get; }
+
         /// <summary>
         /// No isolation
         /// </summary>
         public static ContainerConfiguration DisabledIsolation = new ContainerConfiguration(
                 pathTable: null,
                 redirectedDirectories: CollectionUtilities.EmptyDictionary<ExpandedAbsolutePath, IReadOnlyList<ExpandedAbsolutePath>>(),
-                originalDirectories: CollectionUtilities.EmptyDictionary<AbsolutePath, IReadOnlyList<ExpandedAbsolutePath>>());
+                originalDirectories: CollectionUtilities.EmptyDictionary<AbsolutePath, IReadOnlyList<ExpandedAbsolutePath>>(),
+                isWciFilterEnabled: false);

```

Comment:
Sorry, GitHub can't comment per word... This is for the default value of wci enabled: if we use default should it be true to match existing behavior?

Reply:
Total fail on my part. Fixed.